Week 1/Weekly Practice Problems (Week 1)/1929. Concatenation of Array/dominic.cpp

```diff
@@ -0,0 +1,8 @@
+class Solution {
+public:
+    vector<int> getConcatenation(vector<int>& nums) {
+        int n = nums.size();
+        for (int i = 0; i < n; i++) nums.push_back(nums[i]);
```

Comment:
```suggestion
       nums.insert(nums.end(), nums.begin(), nums.end());
```

Reply:
Yeah, usually vector::push_back in a loop is not so idea since it can cause extra resizing.

if we really want to use push_back, we might want to `reserve` before to minimize resizing